openadapt/events.py

```diff
@@ -25,6 +27,8 @@ def get_events(
     logger.debug(f"raw_action_event_dicts=\n{pformat(raw_action_event_dicts)}")
 
     num_action_events = len(action_events)
```

Comment:
How about:
```
assert action_events, action_events
```

Reply:
I've done assert num_action_events > 0, "No action events found." instead, to produce a message.